dts/arm/96b_carbon.dts

```diff
@@ -27,3 +27,8 @@
 	current-speed = <115200>;
 	status = "ok";
 };
+
+&i2c1 {
+	default-configuration = <0x14>;
```

Comment:
lets not deal with default-configuration right now.  We should re-work how that's done in Zephyr anyways.